### ISSUE TITLE
wrong selected field

### ISSUE THREAD

**Issue by kirill3000** [tokern/data-lineage#52]
For test i select field "page_id" and i expected find all nodes connected only(!) with this field

![image](https://user-images.githubusercontent.com/22203338/123334497-e1a6ca80-d54b-11eb-89de-48694b98cd5f.png)

**Comment by vrajat** [comment]
When `page_id` is selected, too many nodes are selected or too less?

**Comment by kirill3000** [comment]
Too many!
field test.default.filtered_pagecounts.bytes_sent maybe doesn't influence to page_id 

i think it's not correct  mix several tables as inputs and outputs  as far as we don't know how columns influence to each other
![image](https://user-images.githubusercontent.com/22203338/123609383-48580c80-d808-11eb-821d-26ce8f763800.png)

**Comment by vrajat** [comment]
You are right. The root cause is that `LOAD page_lookup_redirect` is downstream of `page_id`. The highlighting algorithm is not intelligent enough and highlights all the columns downstream of the function.
